PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/appservice/AppServiceConfigFormPanelAdvanced.java

```diff
@@ -74,15 +76,16 @@
     private TitledSeparator deploymentTitle;
     private JLabel deploymentLabel;
 
-    public AppServiceConfigFormPanelAdvanced(final Project project) {
+    public AppServiceConfigFormPanelAdvanced(final Project project, final Supplier<T> supplier) {
```

Comment:
is supplier a required part of `AppServiceConfigFormPanelAdvanced`? otherwise IMO, it should not be a constructor parameter

Reply:
@wangmingliang-ms Yes, we need to use the `supplier` to build the Config object